dipy/viz/regtools.py

```diff
@@ -315,7 +315,7 @@ def plot_slices(V, slice_indices=None, fname=None):
         middle slices along each direction are displayed.
     fname : string (optional)
         the name of the file to save the figure to. If None (default), the
-        figure is not saved to disk.
+        the figure is not saved to disk.
```

Comment:
Not needed, duplication with the one above

Reply:
Oh sorry, that didn't see the above "the". Thanks for the feedback.